test/test_context_shape_aggregator.rb

```diff
@@ -89,53 +88,53 @@ def test_prepare_data
   end
 
   def test_sync
-    Timecop.freeze do
```

Comment:
Viewing this with whitespace suppressed will show this is almost entirely the removal of the unnecessary `Timecop.freeze` block